README.org

```diff
@@ -90,42 +76,24 @@ Important note: if ~/home/user/foo.notes~ exists, *will be overwritten*.
 
 *** Uninstalling
 
-Users of
-[[https://github.com/emacscollective/no-littering][no-littering]]
-can take advantage of its packages generated files management.
+Users of [[https://github.com/emacscollective/no-littering][no-littering]] can take advantage of its packages generated files management.
 
 ** keybindings
 
-*** ~C-c  C-a~ (function annotate-annotate)
-    Creates a new annotation for that region.
+*** ~C-c C-a~ (function annotate-annotate)
+Creates a new annotation for that region.
 
-    With no active region, ~C-c C-a~ will create an annotation for the
-    word under point. If point is on an annotated region, ~C-c C-a~
-    will edit that annotation instead of creating a new one. Clearing
-    the annotation deletes them.
+With no active region, ~C-c C-a~ will create an annotation for the word under point. If point is on an annotated region, ~C-c C-a~ will edit that annotation instead of creating a new one. Clearing the annotation deletes them.
 
-    If ~annotate-annotation-confirm-deletion~ is non nil (the default
-    is *nil*) a confirmation action is asked, using ~y-or-n-p~, to the
-    user before actually remove the annotation.
+If ~annotate-annotation-confirm-deletion~ is non nil (the default is *nil*) a confirmation action is asked, using ~y-or-n-p~, to the user before actually remove the annotation.
 
-    If point is the newline character and the customizable variable
-    ~annotate-endline-annotate-whole-line~ is not nil (default is non
-    nil) the whole line is annotated (or the next if the line is
-    empty).
+If point is the newline character and the customizable variable ~annotate-endline-annotate-whole-line~ is not nil (default is non nil) the whole line is annotated (or the next if the line is empty).
 
-    If the line contains a single annotation that cover all the line
-    annotating the newline Will ask to edit the annotation. If
-    ~annotate-endline-annotate-whole-line~ is nil annotating a newline
-    will signal an error.
+If the line contains a single annotation that cover all the line, annotating the newline will ask to edit the annotation. If ~annotate-endline-annotate-whole-line~ is nil annotating a newline will signal an error.
```

Comment:
this is the only non cosmetic fix to readme

Reply:
and now that i reread the changes i think it could be considered cosmetic as well :)